project/src/pages/favorites/favorites.tsx

```diff
@@ -25,101 +25,84 @@ const mapOffersToCity = (arr: Offer[]) =>
   const favoriteOffersMap = mapOffersToCity(favoriteOffers);
 
+  if (offers.length === 0) {
+    return <EmptyFavorites />;
+  }
+
   return (
     <div className="page">
       <Header isAuth />
 
-      <main className={containerCls}>
+      <main className="page__main page__main--favorites">
         <div className="page__favorites-container container">
-          <section className={favoritesCls}>
-            {isEmpty ? (
-              <>
-                <h1 className="visually-hidden">Favorites (empty)</h1>
-                <div className="favorites__status-wrapper">
-                  <b className="favorites__status">Nothing yet saved.</b>
-                  <p className="favorites__status-description">
-                    Save properties to narrow down search or plan your future trips.
-                  </p>
-                </div>
-              </>
-            ) : (
-              <>
-                <h1 className="favorites__title">Saved listing</h1>
-                <ul className="favorites__list">
-                  {Object.keys(favoriteOffersMap).map((cityName) => (
-                    <Fragment key={cityName}>
-                      <li className="favorites__locations-items">
-                        <div className="favorites__locations locations locations--current">
-                          <div className="locations__item">
-                            <a className="locations__item-link" href="/">
-                              <span>{cityName}</span>
-                            </a>
-                          </div>
-                        </div>
-                        <div className="favorites__places">
-                          {favoriteOffersMap[cityName].map((offer) => {
-                            const { id, price, isPremium, isFavorite, title, type, rating, previewImage } = offer;
-                            const link = generatePath('/offer/:id', { id: id.toString() });
+          <section className="favorites">
+            <h1 className="favorites__title">Saved listing</h1>
+            <ul className="favorites__list">
+              {Object.keys(favoriteOffersMap).map((cityName) => (
+                <Fragment key={cityName}>
+                  <li className="favorites__locations-items">
+                    <div className="favorites__locations locations locations--current">
+                      <div className="locations__item">
+                        <a className="locations__item-link" href="/">
+                          <span>{cityName}</span>
+                        </a>
+                      </div>
+                    </div>
+                    <div className="favorites__places">
+                      {favoriteOffersMap[cityName].map((offer) => {
+                        const { id, price, isPremium, isFavorite, title, type, rating, previewImage } = offer;
+                        const link = generatePath('/offer/:id', { id: id.toString() });
```

Comment:
На всякий случай можно так
```suggestion
                        const link = generatePath('/offer/:id', { id: `${id}` });
```